app/static/js/new_post.js

```diff
@@ -0,0 +1,176 @@
+$(function () {
+    // initialize datepickers
+    $('#meeting-date').datepicker({
+        dateFormat: 'mm/dd/yy',
+        maxDate: 0
+    }).keydown(function (e) {
+        // prevent keyboard input except for tab
+        if (e.keyCode !== 9) {
+            e.preventDefault();
+        }
+    });
+
+    $('#next-meeting-date').datepicker({
+        dateFormat: 'mm/dd/yy'
+    }).keydown(function (e) {
+        // prevent keyboard input except for tab
+        if (e.keyCode !== 9) {
+            e.preventDefault();
+        }
+    });
+
+    try {
+        // initialize timepicker plugins
+        $('.timepicker').timepicker({
+            timeFormat: 'h:mm p',
+            interval: 5,
+            minTime: '12:00am',
+            maxTime: '11:59pm',
+            startTime: '9:00am',
+            dynamic: false,
+            dropdown: true,
+            scrollbar: true
+        }).keydown(function (e) {
+            // prevent keyboard input except for allowed keys
+            if (e.keyCode !== 8 && // backspace
+                e.keyCode !== 9 && // tab
+                e.keyCode !== 37 && // left-arrow
+                e.keyCode !== 39 && // right-arrow
+                e.keyCode !== 48 && // 0
+                e.keyCode !== 49 && // 1
+                e.keyCode !== 50 && // 2
+                e.keyCode !== 51 && // 3
+                e.keyCode !== 52 && // 4
+                e.keyCode !== 53 && // 5
+                e.keyCode !== 54 && // 6
+                e.keyCode !== 55 && // 7
+                e.keyCode !== 56 && // 8
+                e.keyCode !== 57 && // 9
+                e.keyCode !== 16 && // Shift
+                e.keyCode !== 65 && // a
+                e.keyCode !== 77 && // m
+                e.keyCode !== 80 && // p
+                e.keyCode !== 186) {// semi-colon
+                e.preventDefault();
+            }
+        });
+
+    }
+    catch (err) {
+        // if one of the forms doesn't have a time field it will throw an error when you try to render it
+        // TODO: find a better way to handle this error
+    }
+
+    // initialize tinymce editor
+    tinymce.init({
+        selector: 'textarea',
+        plugins: 'lists'
+    });
+
+    // set parsley for required fields
+    var requiredFields = ['title',
+                          'meeting-type',
+                          'division',
+                          'meeting-date',
+                          'meeting-location',
+                          'meeting-leader',
+                          'meeting-note-taker',
+                          'start-time',
+                          'end-time',
+                          'attendees',
+                          'tags',
+                          'next-meeting-date',
```

Comment:
If this is a one off, should we be requiring the `next-meeting-*` fields?

Reply:
Will make those optional.